Packages/Status/Sources/Status/Row/Subviews/StatusRowMediaPreviewView.swift

```diff
@@ -279,6 +279,8 @@ public struct StatusRowMediaPreviewView: View {
         .frame(maxWidth: isNotifications ? imageMaxHeight : nil)
         .frame(height: imageMaxHeight)
       }
+      // #965: do not create overlapping tappable areas, when multiple images are shown
+      .contentShape(Rectangle())
```

Comment:
Not sure how, but it sure fixes the issue!

Reply:
it indeed does make sense. I've never checked but the images were just leaking outside of their boundaries.